Gems/AWSGameLift/cdk/aws_gamelift/flex_match/flex_match_configurations.py

```diff
@@ -0,0 +1,34 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+# Matchmaking rule formatted as a JSON string.
+# Comments are not allowed in JSON, but most elements support a description field.
+# For instructions on designing Matchmaking rule sets, please check:
+# https://docs.aws.amazon.com/gamelift/latest/flexmatchguide/match-design-ruleset.html
+RULE_SET_BODY = '{}'

```

Comment:
I think you should give a default and reasonable rule set

Reply:
Talked offline. Will provide a rule set boy with minimum content.